translations/i18nCheck.R

```diff
@@ -40,7 +43,8 @@ msgErrorCheck(rPoError,         "Some translation errors found in po file")
 msgErrorCheck(rEmptyCalls,      "{nrow(rEmptyCalls)} empty gettext call(s) found")
 msgErrorCheck(placeholderData,  "{nrow(placeholderData)} multiple placeholders without index found")
 msgErrorCheck(templateMsgError, "There are numbering errors with multiple placeholders")
-  
+msgErrorCheck(rErrorCalls,      "Don't use `gettext()` if `%` inside, but use `gettextf()` with `%%` instead")
```

Comment:
what is the problem here?

`gettext("95%")`

just works in R?

I think that a lot of code where the confidence or credible interval cannot be set by the user uses something like `gettext("95%")`.

Reply:
These are some error situations that we found in our translation practice that may cause translation loss. see: https://github.com/jasp-stats/jaspQualityControl/pull/286

because the C-level `sprintf()` function from GNU Gettext library is used in backend to format `gettext`, which according to [GNU gettext](https://www.gnu.org/software/gettext/manual/html_node/c_002dformat-Flag.html) will be recognized as a placeholder in some special cases. see also:https://stackoverflow.com/questions/41730770/translating-with-gettext-and-jinja2-and-pyramid